### ISSUE TITLE
Refactor Home template

### ISSUE THREAD

**Issue by hepcat72** [Princeton-LSI-ResearchComputing/tracebase#53]
### Feature Inspiration

There are more model components than compound now.

### Feature Description

Change the home template to compound_list and come up with a home view that links all the existing list views.

### Alternatives Considered

A home view that previews a few records from every other model table.

### Comment

There is no requirement set for a home page on the project roadmap yet.

-----

### Assumptions

none

### Requirements

- separate page for compound list that is the same as the current home page
- Create a very basic home page that links to all the list views for each model component

### Limitations

none

### Affected/Changed Components

- `DataRepo/templates/home.html`
- `DataRepo/templates/compound_list.html`
- `DataRepo/views.py`

#### GUI Change description

Home page changes from a compound list to a table of each database table's list view.

#### Code Change Description *(Pseudocode optional)*

Basically copy the home.html page to compound_list.html and edit the home.html page to link to every list view (currently only the compound list view).

#### Tests

none

**Comment by hepcat72** [comment]
As agreed in the meeting, I am proceeding on this in lieu of rockets.
